src/safety_report_generator.py

```diff
@@ -0,0 +1,139 @@
+from invoke.context import Context
+from datetime import datetime
+
+import json
+import os
+
+TO_BE_DECIDED="To Be Decided"
```

Comment:
Why is `TO_BE_DECIDED` a variable, while `SUCCEEDED`, `FAILED`, and `IGNORED` are all used as string literals?

Reply:
+1 - we should be consistent in styling of these statuses as well. I'm fine if you want to replace with "TBD" since that's a fairly universal acronym.

Also should be spaces in between var, = and value assigned. This would be fixed by running `black -l 120` as @saimidu suggested.

Reply:
Made `TBD` string literal